pkg/sqlstore/impl/system/store.go

```diff
@@ -246,6 +246,10 @@ func (s *SystemStore) GetSchemaByTableName(ctx context.Context, name string) (sq
 		return sqlstore.TableSchema{}, fmt.Errorf("failed to get the table: %s", err)
 	}
 
+	if strings.Contains(strings.ToLower(createStmt), "autoincrement") {
+		createStmt = strings.Replace(createStmt, "autoincrement", "", -1)
+	}
```

Comment:
Uhm, OK, looks fine, but I'm a bit worried we can replace any other stuff with `autoincrement` that shouldn't be replaced. For example, if a column name is `autoincrement`, something bad would happen. 

Of course, this is a very weird border case, but still possible.

I can suggest two options:
1- Simply accept that risk, since it would be very weird if this happened; and wait for a better overall solution, probably related to the other chat about generating "spec compatible" strings.
2- We can do a similar strategy as with `strict`, looking for a more concrete position to replace. For example, maybe look for `primary key autoincrement` (not sure if that would be correct, but you get the idea), and only replace it there. That will probably get rid of this problem while still being a simple-ish solution.

I'm fine with either.